vms/event/views.py

```diff
@@ -220,3 +221,23 @@ def list_events(request):
             'search_result_list': search_result_list
         })
 
+@api_view(['POST'])
+def api_for_vms(request):
```

Comment:
umm.. instead of using function based can we use class based since every view is class based in vms. you can refer this http://www.django-rest-framework.org/tutorial/3-class-based-views/

Reply:
lol nvm.